src/dataset/infrastructure/repositories/DatasetJSDataverseRepository.ts

```diff
@@ -76,10 +95,12 @@ export class DatasetJSDataverseRepository implements DatasetRepository {
             canManageDatasetPermissions: true,
             canDeleteDatasetDraft: true,
             canViewUnpublishedDataset: true
-          },
-          []
+          }, // TODO Connect with JS dataset permissions for privateUrl when it is available in js-dataverse
+          [], // TODO Connect with JS dataset locks for privateUrl when it is available in js-dataverse
+          0, // TODO Connect with JS dataset filesTotalDownloadSize for privateUrl when it is available in js-dataverse
+          0 // TODO Connect with JS dataset filesTotalDownloadSize for privateUrl when it is available in js-dataverse
```

Comment:
Can we create an issue including all these TODOs related to private URLs?

Reply:
I created an issue
- https://github.com/IQSS/dataverse-frontend/issues/266